### PR TITLE
CRYPTO-8: Add the package-info for iniput, output and stream

### PR DESCRIPTION
**Pull request by JkSelf**
This fix https://issues.apache.org/jira/browse/CRYPTO-8

### CODE REVIEW

File: src/main/java/com/intel/chimera/stream/input/package-info.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Input classes
+ */
+package com.intel.chimera.stream.input;
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/input/package-info.java]: please move to apache commons

File: src/main/java/com/intel/chimera/stream/output/package-info.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Output classes
+ */
+package com.intel.chimera.stream.output;
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/output/package-info.java]: please move to apache commons

File: src/main/java/com/intel/chimera/stream/package-info.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Stream classes
+ */
+package com.intel.chimera.stream;
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/package-info.java]: please move to apache commons

**Comment by jerrychenhf** [comment]
+1

**Comment by winningsix** [comment]
LGTM

**Comment by sundapeng** [comment]
Committed to master, thank @JkSelf for your contribution.

**Pull request closed without merging**